_config.yml

```diff
@@ -1,7 +1,8 @@
 name: ROS2 Design
 description: "Distilled design documents related to the ROS 2.0 effort"
 
-baseurl: http://design.ros2.org
+baseurl: ""
+url: http://design.ros2.org
```

Comment:
That's how jekyll is designed to be used, it's mostly to allow github-pages to render properly for project prefixes. 

See: https://jekyllrb.com/docs/github-pages/ and
https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/